helm/prometheus/README.md

```diff
@@ -87,6 +87,34 @@ $ helm install coreos/prometheus --name my-release -f values.yaml
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+
+### Service Monitors
+
+Custom service monitors can be added in values.yaml in the `serviceMonitors` section. Please refere to `values.yaml` for all available parameters.
+
+
+#### Example service monitor
+
+
+This example Service Monitor will monitor applications matching `app: nginx-ingress`. The port `metrics` will be scraped with the path `/merics`. The endpoint will be scraped every 30 seconds.
```

Comment:
Could you add a warning telling the users to check if the labels in the serviceMonitor match the labels defined in the crd from prometheus?

Reply:
Correct "path `/merics" with "path `/metrics" above.

Reply:
@javajon would you like to make a PR to fix that?